src/core/sys/posix/signal.d

```diff
@@ -175,15 +175,28 @@ else version(NetBSD)
 }
 else version (CRuntime_Bionic)
 {
-    enum SIGRTMIN = 32;
-    version(ARM)
-        enum SIGRTMAX = 64;
-    else version(X86)
-        enum SIGRTMAX = 64;
-    else version(MIPS32)
-        enum SIGRTMAX = 128;
-    else
-        static assert(false, "Architecture not supported.");
+    // Switched to calling these functions since Lollipop
+    private extern (C) nothrow @nogc
+    {
+        int __libc_current_sigrtmin();
+        int __libc_current_sigrtmax();
+    }
+
+    @property int SIGRTMIN() nothrow @nogc {
```

Comment:
Could this be a template?

Reply:
It's a straight copy-and-paste of the Glibc block above, as Bionic switched to using the same function-calling scheme as Glibc. If you'd prefer that both blocks use template functions, I can modify them to that, don't care.